### PR TITLE
Bug 1797989: Changes to fix DR scripts with new CEO environment and manifests

### PR DESCRIPTION
**Pull request by retroflexer**
**- What I did**
The changes needed to make DR scripts functional with the introduction of CEO.
**- How to verify it**
Run DR scripts as documented in openshift-docs.
**- Description for the changelog**

Fix DR scripts to make them work in conjunction with CEO.

### CODE REVIEW

**Comment by retroflexer** [comment]
/hold

Will update to reflect a new workflow for restoration.

**Comment by retroflexer** [comment]
/hold cancel

Updated scripts to use latest workflow using restore-pod.yaml on recovery node.

**Comment by hexfusion** [comment]
/lgtm
we need to think about the question above but not blocking.

**Comment by hexfusion** [comment]
cc @runcom @kikisdeliveryservice

File: templates/master/00-master/_base/files/usr-local-bin-etcd-member-add-sh.yaml [hunk 1]
```diff
@@ -44,9 +44,9 @@ contents:
       init
       dl_etcdctl
       backup_manifest
-      DISCOVERY_DOMAIN=$(grep -oP '(?<=discovery-srv=).*[^"]' $ASSET_DIR/backup/etcd-member.yaml )
+      DISCOVERY_DOMAIN=$(grep -oP '(?<=value: "etcd-0\.)[^"]*' $ASSET_DIR/backup/etcd-pod.yaml | head -1)
```
[the remark this answers is not in the document]
Comment (retroflexer) [reply templates/master/00-master/_base/files/usr-local-bin-etcd-member-add-sh.yaml]: At least etcd-0 should be there is what I thought.  In any case, `add` and `recover` is deprecated (hopefully we will delete them). `restore` and `remove` will not need this varirable.

**Comment by sinnykumari** [comment]
error says `does not have enough resources available to fulfill the request.  Try a different zone, or try again later.`

/retest

**Pull request merged**